spec/models/defence_request_spec.rb

```diff
@@ -1,6 +1,24 @@
 require "rails_helper"
 
 RSpec.describe DefenceRequest, type: :model do
+  describe "state_updated_at" do
+    subject { FactoryGirl.create :defence_request }
+
+    it "is set equal created_at when creating" do
+      expect(subject.state_updated_at).to be_within(1.second).of(subject.created_at)
```

Comment:
This value is set within the create transaction if it is > 1sec different it is an indication that something is wrong, not just "slow" tests